hugo/assets/scss/components/header.scss

```diff
@@ -193,10 +193,49 @@
             display: block;
         }
 
+        &__search {
+            position: relative;
+        }
+
         &__toggle {
             display: none;
         }
 
+        &__dropdown {
+            background-color: $c-yellow;
+            border-radius: $b-radius;
+            box-shadow: $shadow--blue;
+            display: none;
+            max-width: $w-site * 0.5;
+            opacity: 0;
+            padding: 2rem $p-gutter--large $p-gutter;
+            position: absolute;
+            right: 0;
+            top: 60px;
+            transform: translate(50%, (-$h-header));
+            transform-origin: right top;
+            transition: opacity .1s, transform .2s;
+            width: $w-site * 0.5;
+
+            &.is-active {
+                display: block;
+            }
+
+            &.is-open {
+                opacity: 1;
+                transform: translate(0%, 0);
```

Comment:
either use 0% or 0 for both, not combine. I think the style-linter want's without a unit so just 0